### PR TITLE
The important information is stored in the .env file

### PR DESCRIPTION
**Pull request by shivansh196**
## Description

 The important information have been stored in the the .env file, which could be accessed using  os.getenv. Also the .env file has been updated in the gitignore file, as this file should not be uploaded on the github repo. This has been done so that as other person who forks the repo is not able to see our private information.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Checklist:

- [x] I have registered myself at [Contrihub website](https://sac.mnnit.ac.in/contrihub).
- [x] My code follows the code style of this project.
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

### CODE REVIEW

**Comment by PriyavKaneria** [comment]
PR is not directed correctly, check from and to
Also make a PR verify request on contrihub

**Review by PriyavKaneria — CHANGES_REQUESTED** [review]
Use more developer friendly ENV variable name like `CONTACT_EMAIL` & `CONTACT_PASSWORD`

**Comment by shivansh196** [comment]
I have made the changes in the variables and in the request ( it is from my fork repo's "secure" branch to contrihub's main ), is it correct?

File: api/app/__init__.py [hunk 1]
```diff
@@ -1,2 +1,9 @@
...
-PSWRD="password_goes_here"
\ No newline at end of file
+
+from dotenv import load_dotenv
+import os
+from fastapi import FastAPI  
+from decouple import config
+
+load_dotenv()
+CONTACT_EMAIL = os.getenv("EMAIL")
```
Comment (PriyavKaneria) [inline api/app/__init__.py:8]: As i mentioned the environment variable itself should be developer friendly so developer understands it when he puts it in ENV.
`os.getenv("CONTACT_EMAIL")`
Code variables are good for maintainers to understand but user first.

File: api/app/api.py [hunk 2]
```diff
@@ -1,22 +1,28 @@
...
 origins = [
     "http://localhost:3000",
     "localhost:3000",
     "*"
 ]
 
 conf = ConnectionConfig(
-    MAIL_USERNAME = EMAIL,
-    MAIL_PASSWORD = PSWRD,
+    CONTACT_EMAIL = EMAIL,
```
Comment (PriyavKaneria) [inline api/app/api.py:24]: Did you run the changes and check? CONTACT_EMAIL is not a parameter for the Connection Config function

File: api/app/__init__.py (hunk 1 above)
Comment (PriyavKaneria) [inline api/app/__init__.py:8]: As i mentioned the environment variable itself should be developer friendly so developer understands it when he puts it in ENV.
`os.getenv("CONTACT_EMAIL")`
Code variables are good for maintainers to understand but user first.

File: api/app/api.py [hunk 3]
```diff
@@ -1,22 +1,28 @@
...
 from fastapi_mail import FastMail, MessageSchema,ConnectionConfig
 import json
 from app import EMAIL, PSWRD
 import os
 
+
+
 app = FastAPI()
 
+EMAIL = os.getenv("EMAIL")
```
Comment (PriyavKaneria) [inline api/app/api.py:14]: If you are using __init__ file to get the env variables you can import them directly, why re-fetch them?

**Review by PriyavKaneria — CHANGES_REQUESTED** [review]
Please check comments. Also do a test run even after making changes if everything works or not. This time the issue was easily identified, it might not be the case everytime.
Thoroughly tested code should be sent for review

**Comment by PriyavKaneria** [comment]
@shivansh196 Any updates on this?

**Comment by PriyavKaneria** [comment]
Closing due to inactivity

**Pull request closed without merging**